packages/react-ui/src/app/routes/platform/billing/dialogs/tasks.tsx

```diff
@@ -90,7 +90,13 @@ export const TasksLimitDialog = ({
                         type="button"
                         variant="transparent"
                         className="absolute right-1 top-1/2 -translate-y-1/2 text-xs"
-                        onClick={() => field.onChange('')}
+                        onClick={() => {
+                          field.onChange('');
+                          form.setError('tasks', {
+                            type: 'manual',
```

Comment:
Why do you need to set it manual? If the typebox allow for null or undefined / empty string then you might not need it at all.

from the user perspective they should be able to remove the limit